diff --git a/locallibs/install.py b/locallibs/install.py
index 4c515a1..e980913 100644
--- a/locallibs/install.py
+++ b/locallibs/install.py
@@ -21,9 +21,9 @@ import os
 import subprocess
 import sys
 
-PYTHON2_EXTRA_PKGS = ["xattr==0.6.4", "pyobjc"]
+PYTHON2_EXTRA_PKGS = []
 
-PYTHON3_EXTRA_PKGS = ["xattr", "pyobjc"]
+PYTHON3_EXTRA_PKGS = []
 
 
 def ensure_pip(framework_path, version):
@@ -34,7 +34,7 @@ def ensure_pip(framework_path, version):
     if not os.path.exists(python_path):
         print("No python at %s" % python_path, file=sys.stderr)
         return
-    cmd = [python_path, "-s", "-m", "ensurepip"]
+    cmd = [python_path, "-s", "/tmp/get-pip.py"]
     print("Ensuring pip is installed...")
     subprocess.check_call(cmd)
 
