diff --git a/python_appimage/utils/docker.py b/python_appimage/utils/docker.py
index c9a0fe7..c48836b 100644
--- a/python_appimage/utils/docker.py
+++ b/python_appimage/utils/docker.py
@@ -22,9 +22,6 @@ def docker_run(image, extra_cmds):
     else:
         raise ValueError('Unsupported Docker image: ' + image)
 
-    log('PULL', image)
-    system(('docker', 'pull', image))
-
     script = [
         'set -e',
         'trap "chown -R {:}:{:} *" EXIT'.format(os.getuid(),
